src/views/activity/components/transaction-card/transaction-card.styles.ts

```diff
@@ -76,7 +76,7 @@ const useTransactionCardStyles = createUseStyles((theme: Theme) => ({
     justifyContent: "space-between",
     alignItems: "center",
   },
-  finaliseButton: {
+  finalizeButton: {
```

Comment:
`finalise` is more correct than `finalize`. This was corrected by Jenna in the wallet.

Reply:
Really? I thought it was the other way around.

Reverted!